required/firstaid/latex2e-first-aid-for-external-files.dtx

```diff
@@ -717,6 +717,34 @@
             {[\@tempb][\arabic{page}][\cref@result]\thepage}{}{}{}}}% <------- five
           \@esphack}%
           } 
+%    \end{macrocode}
+% \changes{v1.1g}{2024/10/16}{update cleveref data in label hook}
+% cleveref patches and redefines \cs{refstepcounter} so that a call updates its data.
+% This fails if like e.g. in \pkg{longtable} the counter is stepped with 
+% \cs{@kernel@refstepcounter}. We therefore move the data setup into the label hook.
+% As the hook is in a group we have to smuggle the data out of it.
+% \url{https://tex.stackexchange.com/a/722909/2388} and issue \#1393
```

Comment:
or perhaps rethink the the  hook position behavior of "label"?

Reply:
I could change that as part of the PR.